src/component/Table/Table.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import './Table.css';
+import numeral from 'numeral';
+
+const Table = ({countries}) => {
+    return (
+        <div className = "table">
```

Comment:
You are using tr and td without the table element.

```suggestion
        <table className = "table">
```

Reply:
Fixed this in the latest commit.